company/okrs/2020_q1.md

```diff
@@ -0,0 +1,115 @@
+1. **Distribution: Build features that accelerate customers through pipeline**
+   1. Distribution KR: Shorten the time from "intent to try Sourcegraph" to "whole org is using Sourcegraph" (onboarding experience)
+   1. Distribution KR: Ensure new features (LSIF, GitLab Native Integration, Automation) are accessible and easily tryable by customers once they go GA
+   1. Distribution KR: Surface information to site admins and managers to encourage expansion/upsell
+
+1. **Code intelligence: LSIF is adopted by customers**
+   1. Code intelligence KR: At least three customers are using LSIF on their primary repositories
+   1. Code intelligence KR: At least one customer is using LSIF across all of their code in languages for which there exists an LSIF indexer
+   1. Code intelligence KR: Start tracking the performance (user interaction to result) of hover tooltips, go to definition, and find references
+
+1. **Code intelligence: Increase user confidence in code intelligence provided by LSIF**
+   1. Code intelligence KR: Code intelligence UI (tooltips, references panel, etc.) indicates whether or not results are precise
+   1. Code intelligence KR: Fall back to fuzzy if the file has been modified since the last LSIF upload
+   1. Code intelligence KR: Make use of LSIF data when the current line is not part of the diff since the last LSIF upload
+
+1. **Code intelligence: Make search better using precise code intel data**
+   1. Code intelligence KR: Implement search within dependencies and/or dependents of current repo
+   1. Code intelligence KR: Implement searching for defs/refs by symbol name
+
+1. **Web: Make users aware of advanced search features**
+   1. Web KR: Start tracking time to first search and search filters per query in pings
+   1. Web KR: Shorten time to first search
+   1. Web KR: Increase average number of search filters per query
+
+1. **Web: Automation is adopted by customers**
+   1. Web KR: At least four customers are using campaigns in production
+   1. Web KR: All campaign types have been used by customers in production
+
+1. **Core Services: Build features that accelerate customers through pipeline**
+   1. Core Services KR: Shorten the time from "intent to try Sourcegraph" to "whole org is using Sourcegraph" (onboarding experience)
+   1. Core Services KR: Ensure new features (Automation, ACLs, etc) are accessible and easily tryable by customers once they go GA
+
+1. **Core Services: Search has feature parity with OpenGrok**
+   1. Core Services KR: Implement AND query support
+   1. Core Services KR: Implement OR query support
+   1. Core Services KR: Implement NOT query support
+
+1. **Core Services: Existing customers are cared for**
+   1. Core Services KR: Each iteration work plan includes prioritised customer issues.
+   1. Core Services KR: Each team mate works on relevant customer issues every iteration.
+
+1. **Core Services: Automation is polished and adopted by customers**
```

Comment:
@lguychard @tsenart Here's the complete list of key results that we came up with in today's Core Services sync:

```
1. Core Services KR: Document Automation and write a blog post about it.
1. Core Services KR: Automation is enabled by default and not behind a feature flag.
1. Core Services KR: Run 3 user tests with Sourcegraph colleagues, collect issues and fix them
1. Core Services KR: 3 different customers have together run at least 10 campaigns.
```

Feel free to edit/remove/add to this.

I'm also still unsure how to get the features in this list or whether they should be in there at all, since we're going to work on them anyway.